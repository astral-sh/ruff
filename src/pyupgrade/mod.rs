mod checks;
pub mod fixes;
pub mod plugins;
pub mod types;
