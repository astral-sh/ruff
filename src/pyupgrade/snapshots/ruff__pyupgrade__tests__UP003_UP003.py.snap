---
source: src/pyupgrade/mod.rs
expression: checks
---
- kind:
    TypeOfPrimitive: Str
  location:
    row: 1
    column: 0
  end_location:
    row: 1
    column: 8
  fix:
    content: str
    location:
      row: 1
      column: 0
    end_location:
      row: 1
      column: 8
  parent: ~
- kind:
    TypeOfPrimitive: Bytes
  location:
    row: 2
    column: 0
  end_location:
    row: 2
    column: 9
  fix:
    content: bytes
    location:
      row: 2
      column: 0
    end_location:
      row: 2
      column: 9
  parent: ~
- kind:
    TypeOfPrimitive: Int
  location:
    row: 3
    column: 0
  end_location:
    row: 3
    column: 7
  fix:
    content: int
    location:
      row: 3
      column: 0
    end_location:
      row: 3
      column: 7
  parent: ~
- kind:
    TypeOfPrimitive: Float
  location:
    row: 4
    column: 0
  end_location:
    row: 4
    column: 8
  fix:
    content: float
    location:
      row: 4
      column: 0
    end_location:
      row: 4
      column: 8
  parent: ~
- kind:
    TypeOfPrimitive: Complex
  location:
    row: 5
    column: 0
  end_location:
    row: 5
    column: 8
  fix:
    content: complex
    location:
      row: 5
      column: 0
    end_location:
      row: 5
      column: 8
  parent: ~

